src/yb/tools/yb-admin_cli.cc

```diff
@@ -526,6 +526,19 @@ void ClusterAdminCli::RegisterCommandHandlers(ClusterAdminClientClass* client) {
         return Status::OK();
       });
 
+  Register(
+      "flush_sys_catalog", 
+      "[timeout_in_seconds] (default 20)",
+      // " <table> [timeout_in_seconds] (default 20)"
+      [client](const CLIArguments& args) -> Status {
+        int timeout_secs = 20;
```

Comment:
how do we get the user-provided timeout?

Reply:
@robertsami  Hi, for the flush_table, it uses VERIFY_RESULT(GetTimeoutAndAddIndexesFlag(
          args.begin() + 1, args.end())); to bind timeout_secs and add_index. I can use the same one for binding. Then do I need to add add_indexes for flush_sys_catalog? What is add_indexes used for?